variables.cmake

```diff
@@ -89,7 +89,7 @@ if (WIN32)
 elseif (APPLE)
     set(LIBSDIR "Frameworks")
 elseif (UNIX)
-    set(LIBSDIR "lib/${CMAKE_C_LIBRARY_ARCHITECTURE}")
+    set(LIBSDIR "${CMAKE_INSTALL_LIBDIR}")
```

Comment:
I need to check if this works on the Raspberry Pi and in general on Debian package creation

Reply:
On Fedora this gives `lib64`.

The CMake implementation explicitly mentions that it works different for Debian and does so here: https://gitlab.kitware.com/cmake/cmake/-/blob/v3.26.4/Modules/GNUInstallDirs.cmake?ref_type=tags#L276

But yes, testing it is the best thing to do (I don't have a RasPi, so I cannot do that).

Reply:
Just checked on Ubuntu

```
---> CMAKE_C_LIBRARY_ARCHITECTURE x86_64-linux-gnu
---> CMAKE_INSTALL_LIBDIR lib/x86_64-linux-gnu
```

So this is not gonna work on Debian-like systems.
How can we solve this?

Reply:
EDIT: just checked on the Raspberry Pi and this is the result

```
---> CMAKE_C_LIBRARY_ARCHITECTURE aarch64-linux-gnu
---> CMAKE_INSTALL_LIBDIR lib
```

So, no joy :cry: